doc/nrf/libraries/networking/nrf_provisioning.rst

```diff
@@ -23,20 +24,30 @@ Configuration
 
 To enable the library, set the :kconfig:option:`CONFIG_NRF_PROVISIONING` Kconfig option to ``y``.
 
-Configuration options for transport protocol
-============================================
-
-Currently, HTTP is the only supported transport protocol.
-
-* :kconfig:option:`CONFIG_NRF_PROVISIONING_SYS_INIT` - Initializes the client in the system initialization phase
+* :kconfig:option:`CONFIG_NRF_PROVISIONING_AUTO_INIT` - Initializes the client in the system initialization phase
 * :kconfig:option:`CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG` - Root CA security tag for the Provisioning Service
+* :kconfig:option:`CONFIG_NRF_PROVISIONING_INTERVAL_S` - Maximum provisioning interval in seconds
+* :kconfig:option:`CONFIG_NRF_PROVISIONING_WITH_CERT` - Provisions the root CA certificate to the security tag if the tag is empty
```

Comment:
Which root CA? AWS?

Reply:
Is it only the multi-service sample with CoAP that needs both?

Reply:
Provisioning uses AWS root CA for HTTP communication. CoAP connection requires different root CA

Reply:
Yeah. In the upcoming guide for the multi-service sample, the user needs to inject both the AWS root CA _and_ CoAP CA cert, one after the other, into the same slot.

Adding the --coap argument to the [device credentials installer](https://github.com/nRFCloud/utils/blob/master/python/modem-firmware-1.3+/README.md#create-ca-cert) script also injects both at the same time.

Reply:
OK, what was confusing about this was that the sec tags involved aren't the default. I'm still not sure that's clear enough.

Think I need more info from QA about what this actually looks like in practice, because they've been trying it out.

Reply:
My plan for now is to address, if needed, the specific security tags for these certs in the context of a secure provisioning guide (with options for CoAP and HTTP). 

No further changes needed here.